### PR TITLE
Colour parsing using Lodash and Colornames (Light The Bridge)

### PR DESCRIPTION
**Pull request by WJXHenry**
Solves issue #69 

Uses the [Colornames](https://www.npmjs.com/package/colornames) npm package to parse colours from the tweet title.

### CODE REVIEW

File: src/controllers/light-the-bridge.js [hunk 1]
```diff
@@ -25,11 +29,15 @@ module.exports = async function(req, res) {
...
     ...{
       created_at: new Date().toISOString(),
-      color_description: '',
-      color1: '',
-      color2: '',
-      color3: '',
-      color4: '',
+      color_description: colors
+        .map(function(element) {
+          return _.capitalize(element.color)
```
Comment (j-rewerts) [inline src/controllers/light-the-bridge.js]: It's probably okay to return lower case color names.

File: src/utils/color-parse.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var _ = require('lodash')
+var toHex = require('colornames')
+
+function parseColors(string_to_parse) {
+  const words = _.words(_.toLower(string_to_parse))
```
Comment (j-rewerts) [inline src/utils/color-parse.js]: I think based on the amount we're using lodash, it'd be best to just use JS native functions.

Comment (j-rewerts) [reply src/utils/color-parse.js]: In this case use split to split the string into words. Then the toLowerCase function.

File: package.json [hunk 3]
```diff
@@ -30,10 +30,13 @@
   "homepage": "https://github.com/j-rewerts/IFTTT-Edmonton#readme",
   "dependencies": {
     "body-parser": "^1.18.3",
+    "colornames": "^1.1.1",
     "express": "^4.16.4",
+    "lodash": "^4.17.11",
```
Comment (j-rewerts) [inline package.json]: So the current uses of this are for capitalizing and lowercasing words it looks like. We can do that using regular old JS.

File: package.json [hunk 4]
```diff
@@ -30,10 +30,13 @@
...
     "redis": "^2.8.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.7",
+    "string-to-color": "^2.0.1",
```
Comment (j-rewerts) [inline package.json]: Are we using this?

Comment (WJXHenry) [reply package.json]: Nope, I just forgot to take that out

File: src/controllers/light-the-bridge.js (hunk 1 above)
Comment (WJXHenry) [reply src/controllers/light-the-bridge.js]: I left it as capitalized, but if you prefer it all lowercase I can change it too

**Pull request merged**